extension/scripts/features/jail-filter/ttJailFilter.js

```diff
@@ -108,6 +108,13 @@
 		filterContent.appendChild(scoreFilter.element);
 		localFilters["Score Filter"] = { getStartEnd: scoreFilter.getStartEnd, updateCounter: scoreFilter.updateCounter };
 
+		const bailCostFilter = createTextbox({ type: "number", description: "Bail Cost less than", min: 0 });

```

Comment:
I suggest the description as "Maximum Bail Cost". It is lesser verbose IMO.

Reply:
made the changes